docs/java-rest/high-level/security/delete-role-mapping.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[java-rest-high-security-delete-role-mapping]]
+=== Delete Role Mapping API
+
+[[java-rest-high-security-delete-role-mapping-execution]]
+==== Execution
+Deletion of a role mapping can be performed using the `security().deleteRoleMapping()`
+method:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/SecurityDocumentationIT.java[delete-role-mapping-execute]
+--------------------------------------------------
+
+[[java-rest-high-security-delete-role-mapping-response]]
+==== Response
+The returned `DeleteRoleMappingResponse` contains a single field, `found`. If the mapping
+is successfully found and deleted, the request returns true. Otherwise, found is set to false.
```

Comment:
> If the mapping is successfully found and deleted, the request returns true. Otherwise, found is set to false.

Maybe change to 

> If the mapping is successfully found and deleted, found is set to true. Otherwise, it is set to false.